test/e2e/page-objects/CartOverview.js

```diff
@@ -0,0 +1,91 @@
+import Base from './Base';
+
+const SCREEN_SELECTOR = '#cart_contents_container';
+
+class CartOverview extends Base {
```

Comment:
@wswebcreation hmmm, CartOverview? A bit confused regarding what this page actually is representing?

Reply:
Tnx @nadvolod , you are right, but how would you like to name this Class? It's the overview of whats in the cart, but open for other suggestions

Reply:
@wswebcreation are we talking about this page here: https://www.evernote.com/l/AWSnghfA1jBLs5mekRougP_8LqjDtd7k6WE
If that's the case, that's just the `ShoppingCartPage` to me. Even the URL says that.

Reply:
Yep, but I personally _hate_ to add page to it, it's already a page object

Reply:
fwiw I used to agree with you @wswebcreation , but I've started always adding "Page", "Data", and "API" to the classes I create, even when they are in a folder of that name, and subclassed to a base object of that name, just so that when I look at the name in my IDE tab I know exactly which file it is.

Reply:
You are right, updated all names now and added the postfix Page